content/SCALETutorials/Instances/_index.md

```diff
@@ -19,554 +19,633 @@ keywords:
 
-### Changing the Device Order
+2. Configure the **CPU & Memory** settings.
 
-1. Stop the VM if it is running, then click **Devices** to open the list of devices for the selected VM
-2. Click **Edit**.
-3. Enter the number that represents where in the boot sequence you want this device to boot in the **Devices Order** field.
-   The lower the number, the higher the device is in the boot sequence.
-4. Click **Save**.
-5. Restart the VM.
+   Compare the recommended specifications for the guest operating system with your available host system resources when allocating CPU and memory resources.
 
-### Adding a CD-ROM Device
+   {{< trueimage src="/images/SCALE/Virtualization/CreateVMCPUandMemory.png" alt="CPU & Memory - VM" id="CPU & Memory - VM" >}}
 
-Select **CD-ROM** as the **Device Type** on the **Add Device** screen and set a boot order.
+   For VMs, CPU and memory configuration is required.
 
-1. Stop the VM if it is running, then click **Devices**.
-2. Click **Add** and select **CD-ROM** from the **Device Type** dropdown list.
+   {{< include file="/static/includes/InstanceCPUMemoryProcedure.md" >}}
 
-   {{< trueimage src="/images/SCALE/Virtualization/VMAddDeviceCDROM.png" alt="Devices Add CD-ROM Type" id="Devices Add CD-ROM Type" >}}
+3. Configure the **Disks** settings to mount storage volumes for the VM.
+   For VMs, you must specify the I/O bus and size of the root disk.
+   You can also mount one or more existing zvol(s) as additional storage, if needed.
+   See [Storage volume types](https://linuxcontainers.org/incus/docs/main/explanation/storage/#storage-volume-types) from Incus for more information.
 
-3. Specify the mount path.
-   Click on the <i class="fa fa-caret-right" aria-hidden="true"></i> to the left of **/mnt** and at the pool and dataset levels to expand the directory tree. Select the path to the CD-ROM device.
-4. Specify the boot sequence in **Device Order**.
-5. Click **Save**.
-6. Restart the VM.
+   {{< trueimage src="/images/SCALE/Virtualization/CreateInstanceDisksVM.png" alt="Disks - VM" id="Disks - VM" >}}
 
-### Adding a NIC Device Type
+   a. Choose the root disk I/O bus type that best suits system needs. Options are:
 
-Select **NIC** in the **Device Type** on the **Add Device** screen to add a network interface card for the VM to use.
+      - **NVMe** – Ideal for high-performance storage with faster read and write speeds.
 
-1. Stop the VM if it is running, then click **Devices**.
-2. Click **Add** and select **NIC** from the **Device Type** dropdown list.
+      - **Virtio-BLK** – Efficient for virtualized environments, offering direct block device access with lower overhead.
 
-   {{< trueimage src="/images/SCALE/Virtualization/VMAddDeviceNIC.png" alt="Devices Add NIC" id="Devices Add NIC" >}}
+      - **Virtio-SCSI** – Flexible and scalable, supporting advanced features like hot-swapping and multiple devices.
 
```

Comment:
To make this read less like UI ref, you can do something like this:
a. Choose the root disk I/O bus type that best suits system needs.
    
   Use **NVMe** for high-performance storage with faster read/write speeds.

   Use **Virtio-BLK** for efficient virtualized environments that need direct block device access with lower overhead.

   Use **Virtio-SCSI for flexible and scalable storage that supports advanced features like hot-swapping and multiple devices.